### PR TITLE
[Chore] Refactor Helper files and functions

### PR DESCRIPTION
**Pull request by jacobtyq**
Description
---
This PR is to refactor the helper files and functions for better code maintainability and readability. This PR also includes a `__tests__` folder to hold tests.
- Refactor helpers to use meaningful function names (e.g. `ucFirstAllWords` vs `capitalizeFirstLetter`)
- Remove Helpers/index.js and export directly from helpers.js
- Update Nearby.js and SearchAll.js with new imports and function names
- Create `__tests__/helpers.test.js` to test the functions in Helpers

Would suggest shifting `distance_calc()` into Helpers file as well after this PR is approved

### CODE REVIEW

File: src/Helpers/helpers.js [hunk 1]
```diff
@@ -14,14 +14,20 @@ export function cmpStr (str1, str2) {
...
 	}
-	return pieces.join(" ");
+	return words.join(" ");
 }
+
+
+export default {
+	compareString,
+	capitalizeFirstLetter
+}
```
Comment (achrinza) [inline src/Helpers/helpers.js:33]: Out of scope but I think it's a good discussion point:
IMO we should move away from default exports as they may become a pain point when refactoring code.

File: __tests__/helpers.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Helpers from "../src/Helpers/helpers";
```
[the remark this answers is not in the document]
Comment (jacobtyq) [reply __tests__/helpers.test.js:6]: Sure, updated. Might want to include this into the README or CONTRIBUTING file

[the remark this answers is not in the document]
Comment (achrinza) [reply __tests__/helpers.test.js:6]: > Might want to include this into the README or CONTRIBUTING file

Yup, we should. I've opened #32 to track this issue.

**Comment by jacobtyq** [comment]
Sure, will wait for https://github.com/limyifan1/hawkercentral/pull/31 to be merged then

**Comment by joycesin** [comment]
thank you both for your efforts! have merged #31

**Comment by jacobtyq** [comment]
Have rebased my branch

**Comment by joycesin** [comment]
thanks everyone, couldn't do this without you guys 👍

**Pull request merged**